etna/whatson/models.py

```diff
@@ -0,0 +1,621 @@
+        verbose_name=_("end date"),
+        null=True,
+        blank=True,
+        editable=False,
+    )
+
+    description = RichTextField(
+        verbose_name=_("description"),
+        blank=True,
+        help_text=_("A description of the event."),
+    )
+
+    useful_info = RichTextField(
+        verbose_name=_("need to know"),
+        blank=True,
+        help_text=_("Useful information about the event."),
+    )
+
+    target_audience = RichTextField(
+        verbose_name=_("who it's for"),
+        blank=True,
+        help_text=_("Info about the target audience for the event."),
+    )
+
+    # Venue information
+    venue_type = models.CharField(
+        max_length=15,
+        verbose_name=_("venue type"),
+        null=False,
+        blank=False,
+        choices=VenueType.choices,
+        default=VenueType.IN_PERSON,
+    )
+
+    venue_website = models.URLField(
+        max_length=255,
+        verbose_name=_("venue website"),
+        null=True,
+        blank=True,
+        help_text=_("The website for the venue."),
+    )
+
+    venue_address = RichTextField(
+        verbose_name=_("venue address"),
+        blank=True,
+        help_text=_("The address of the venue."),
+    )
+
+    venue_space_name = models.CharField(
+        max_length=255,
+        verbose_name=_("venue space name"),
+        blank=True,
+        help_text=_("The name of the venue space."),
+    )
+
+    video_conference_info = RichTextField(
+        verbose_name=_("video conference info"),
+        blank=True,
+        help_text=_("Useful information about the video conference."),
+    )
+
+    # Booking information
+    booking_type = models.CharField(
+        max_length=20,
+        verbose_name=_("booking type"),
+        blank=False,
+        choices=BookingType.choices,
+        default=BookingType.DROP_IN,
+        editable=False,
+    )
+
+    registration_url = models.URLField(
+        max_length=255,
+        verbose_name=_("registration url"),
+        null=True,
+        blank=True,
+        editable=False,
+    )
+
+    registration_cost = models.IntegerField(
+        verbose_name=_("registration cost"),
+        null=True,
+        blank=True,
+        editable=False,
+    )
+    # The three fields above will be brought in from the API when we have it.
+
+    registration_info = RichTextField(
+        verbose_name=_("registration info"),
+        blank=True,
+        help_text=_("Additional information about how to register for the event."),
+    )
+
+    contact_info = RichTextField(
+        verbose_name=_("contact info"),
+        blank=True,
+        help_text=_("Information about who to contact regarding the event."),
+    )
+
+    # Promote tab
+    short_title = models.CharField(
+        max_length=50,
+        verbose_name=_("short title"),
+        help_text=_(
+            "A short title for the event. This will be used in the event listings."
+        ),
+        blank=False,
```

Comment:
`False` is always the default value for `blank`, so you don't need to specify that here

```suggestion
```